terraform/workspace_variables/app_config.yml

```
@@ -6,6 +6,9 @@ default: &default
   SETTINGS__FEATURES__SIGNIN_BY_EMAIL: false
   SETTINGS__FEATURES__SIGNIN_INTERCEPT: false
   SETTINGS__ROLLOVER: false
+  SETTINGS__FEATURES__MAINTENANCE_MODE__ENABLED: false
+  SETTINGS__FEATURES__MAINTENANCE_MODE__TITLE: "This service will be unavailable from Thursday 4 March at 17:00 to Friday 5 March at 09:00, while we carry out maintenance."
+  SETTINGS__FEATURES__MAINTENANCE_MODE__BODY: "Find teacher training courses will still be available during this time. If you have any questions, contact us at becomingateacher@digital.education.gov.uk"
```

```suggestion
  SETTINGS__FEATURES__MAINTENANCE_MODE__TITLE: "This service will be unavailable from Thursday 4 March at 17:00 to Friday 5 March at 09:00, while we carry out maintenance."
  SETTINGS__FEATURES__MAINTENANCE_MODE__BODY: "Find teacher training courses will still be available during this time. If you have any questions, contact us at becomingateacher@digital.education.gov.uk"
```
thanks but not sure if TITLE and BODY are required here..

---

I think it will be good if we can set these from the ENV in case we need to change the message if we have any problems. e.g. if we're not ready at 9am.